ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -417,8 +417,9 @@ protected int determineActiveToolState() {
     }
 
     private int determineLanguageState(final int languageIndex) {
-        if (mManifests.get(languageIndex) != null) {
-            return STATE_LOADED;
+        Manifest manifest = mManifests.get(languageIndex);
+        if (manifest != null) {
+            return Tool.Type.TRACT.equals(manifest.getType()) ? STATE_LOADED : STATE_NOT_FOUND;
```

Comment:
`Type` is an enum, so you can do `manifest.getType() == Type.TRACT`. Also this is currently the wrong `Type` enum you are comparing against, you need to make the Manifest.Type enum definition public so you can access it here.